how-to/software/automatic-updates.md

```diff
@@ -1,82 +1,383 @@
 
-To install the `apticron` package, enter the following command in a terminal:
+### Manual steps required
+While Ubuntu updates rarely require manual steps to complete an upgrade (at most a reboot can be required), it could be plausible that other applications require some manual steps after or before an update is applied. If that is the case, and if such steps cannot be safely automated, then maybe *unattended-upgrades* should be disabled on such systems. But do make an effort to consider blocklisting such packages instead, if they are known to trigger such manual steps. In that case, the system can still benefit from all the other upgrades that might become available.
+
+### Too much of a risk
+Even with all the care in the world, applying updates to a running system comes with risk. Ubuntu believes that risk to be less than the risk of NOT applying a security update, which is why *unattended-upgrades* will apply security updates by default. But for some specific systems, the risk vs benefit equation might favor staying put and not applying an update unless specifically requested.
+
+Always keep in mind, however, that specific packages can be blocked from receiving updates. For example, if a particular system runs a critical application that could break if certain libraries on the system are updated, then perhaps an acceptable compromise is to block these library packages from receiving upgrades, instead of disabling the whole feature.
+
+### Fleet management
+The *unattended-upgrades* feature is helpful, does its job, and even sends out reports. But it's not meant to be a replacement for fleet management software. If a large number of Ubuntu systems needs to be kept updated, other solutions are better suited for the job. Such large deployments usually come with much stricter and wider requirements, like:
+ * compliance reports: How many systems are up-do-date, how many are still behind, for how long has a system been exposed to a known vulnerability, etc.
+ * maintenance windows: Different systems might require different maintenance windows. Some can be updated anytime, others only on weekends, etc.
+ * canary rollouts: The ability to rollout updates to an initial group of systems, and over time increase the number of systems which will receive the update.
+
+An example of such a Fleet Management software for Ubuntu systems is [Landscape](https://ubuntu.com/landscape).
+
+
+## Testing and troubleshooting
+It's possible to test some configuration changes to *unattended-upgrade* without having to wait for the next time it would run. The `unattended-upgrade` tool has a [manual page](https://manpages.ubuntu.com/manpages/noble/man8/unattended-upgrade.8.html) which explains all its command-line options. Here are the most useful ones for testing and troubleshooting:
```

Comment:
```suggestion
It's possible to test some configuration changes to `unattended-upgrade` without having to wait for the next time it would run. The `unattended-upgrade` tool has a [manual page](https://manpages.ubuntu.com/manpages/noble/man8/unattended-upgrade.8.html) that explains all its command-line options. Here are the most useful ones for testing and troubleshooting:
```